docs/cli/nrfu.md

```diff
@@ -145,6 +145,36 @@ anta nrfu --tags LEAF json
 ```
 ![$1anta nrfu json results](../imgs/anta-nrfu-json-output.png){ loading=lazy width="1600" }
 
+## Performing NRFU and saving results in a Markdown file
+
+The `md-report` command in NRFU testing generates a comprehensive Markdown report containing various sections, including detailed statistics for devices and test categories. By default, this command saves all test results from the current run in the Markdown file. However, you can choose to include only failed tests by using the `--only-failed-tests` option in the command line interface.
+
+This feature is particularly useful for comparing reports from multiple runs, allowing you to quickly identify and track issues across your network over time. By focusing on failed tests, you can efficiently pinpoint areas that require attention or have shown improvement between different test executions.
+
+### Command overview
+
+```bash
+anta nrfu md-report --help
+
+Usage: anta nrfu md-report [OPTIONS]
+
+  ANTA command to check network state with Markdown report.
+
+Options:
+  --md-output FILE     Path to save the report as a Markdown file. It only
+                       saves test results and not the output from the --group-
+                       by option.  [env var: ANTA_NRFU_MD_REPORT_MD_OUTPUT;
```

Comment:
this should not be there

Reply:
Fixed in 30c8d7e418e1f273e882bb1dbdc3ac8e7151a4dd